data/data/bootstrap/baremetal/files/etc/NetworkManager/conf.d/99-baremetal.conf

```diff
@@ -1,3 +1,14 @@
 [connection]
 ipv6.dhcp-duid=ll
 ipv6.dhcp-iaid=mac
+
+{{if .PlatformData.BareMetal.ExternalStaticIP}}
+type=ethernet
+interface-name=ens3
+[ethernet]
+[ipv4]
```

Comment:
You can probably determine v4/v6 using the existing `UseIPv6ForNodeIP` variable added via #4756